app/server/middleware/identityMiddleware.ts

```diff
@@ -132,106 +133,101 @@ const redirectOrCustomStatusCode = (
   statusCode?: number
 ) =>
   statusCode
-    ? res
-        .status(statusCode)
-        .header("Location", redirectURL)
-        .send()
+    ? res.status(statusCode).header("Location", redirectURL).send()
     : res.redirect(redirectURL);
 
 export const getCookiesOrEmptyString = (req: express.Request) =>
   req.header("cookie") || "";
 
 export const withIdentity: (
   statusCodeOverride?: number
-) => express.RequestHandler = (statusCodeOverride?: number) => (
-  req: express.Request,
-  res: express.Response,
-  next: express.NextFunction
-) => {
-  const errorHandler = (message: string, detail?: any) => {
-    handleAwsRelatedError(message, detail);
-    res.sendStatus(500); // TODO maybe server side render a pretty response
-  };
-
-  const useRefererHeaderForManageUrl = !!statusCodeOverride;
-
-  idapiConfigPromise
-    .then(idapiConfig => {
-      if (idapiConfig) {
-        fetch(
-          url.format({
-            protocol: "https",
-            host: idapiConfig.host,
-            pathname: "auth/redirect"
-          }),
-          {
-            headers: {
-              "X-GU-ID-Client-Access-Token":
-                "Bearer " + idapiConfig.accessToken,
-              [X_GU_ID_FORWARDED_SCOPE]:
-                req.header(X_GU_ID_FORWARDED_SCOPE) ||
-                getScopeFromRequestPathOrEmptyString(req.path),
-              Cookie: getCookiesOrEmptyString(req)
+) => express.RequestHandler =
+  (statusCodeOverride?: number) =>
+  (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    const errorHandler = (message: string, detail?: any) => {
+      handleAwsRelatedError(message, detail);
+      res.redirect("/maintenance");
```

Comment:
This line is the only code change in this file, replacing `sendStatus(500)` with a redirect to the maintenance page. The remainder of the diff is due to Prettier reformatting.